### PR TITLE
feat: add wcp check for audit logs

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Adds ability to activate and deactivate `Audit Logs` module via `WCP`.

## How Has This Been Tested?
Manual

## Documentation
None

### CODE REVIEW

File: packages/app-audit-logs/src/plugins/permissionRenderer/index.tsx [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+                >
+                    <AuditLogsPermissionsComponent {...props} />
+                </AccordionItem>
+            );
+        }
+    };
 };
 
-export default plugin;
+export const AuditLogsPermissions: React.FC = () => {
```
Comment (adrians5j) [inline packages/app-audit-logs/src/plugins/permissionRenderer/index.tsx:32]: Why this approach? Why not just export plugin like we have it with other?

File: apps/admin/src/plugins/index.ts [hunk 2]
```diff
@@ -21,10 +20,6 @@ plugins.register([
      * Headless CMS app.
      */
     headlessCmsPlugins,
-    /**
-     * Audit Logs app.
-     */
-    auditLogsPlugins,
```
Comment (adrians5j) [inline apps/admin/src/plugins/index.ts:27]: Why was this removed? 🤔

**Review by adrians5j — CHANGES_REQUESTED** [review]
Left two questions.

File: packages/app-audit-logs/src/plugins/permissionRenderer/index.tsx (hunk 1 above)
Comment (neatbyte-vnobis) [reply packages/app-audit-logs/src/plugins/permissionRenderer/index.tsx:32]: I've used the same approach as APW package, that you've suggested to use as reference. [Here](https://github.com/webiny/webiny-js/blob/next/packages/app-apw/src/plugins/permissionRenderer/index.tsx#L27) is APW permission settings plugin and it is registered based on [this](https://github.com/webiny/webiny-js/blob/next/packages/app-apw/src/index.tsx#L45) condition.

File: apps/admin/src/plugins/index.ts (hunk 2 above)
Comment (adrians5j) [reply apps/admin/src/plugins/index.ts:27]: Question here remains.

**Review by adrians5j — CHANGES_REQUESTED** [review]
One question remanins.

File: apps/admin/src/plugins/index.ts (hunk 2 above)
Comment (neatbyte-vnobis) [reply apps/admin/src/plugins/index.ts:27]: Please refer to [this](https://github.com/webiny/webiny-js/pull/3580#discussion_r1358301392) answer. Plugin is now registered inside the package based on condition.

Comment (adrians5j) [reply apps/admin/src/plugins/index.ts:27]: Srry, thanks!

**Pull request merged**